config/jobs/cert-manager/cert-manager/cert-manager-periodics.yaml

```diff
@@ -339,59 +354,9 @@ periodics:
         options:
           - name: ndots
             value: "1"
-  # TODO: find a permanent home for the AWS periodics and reinstate this job
-  # - name: aws-tests
-  #   interval: 48h
-  #   agent: kubernetes
-  #   decorate: true
-  #   extra_refs:            # Periodic job doesn't clone any repo by default, needs to be added explicitly
```

Comment:
note: Losing this commented out test seems acceptable; it'll stay in git history and we've not touched this test since it was written